ui/src/components/system/UsersGroups.vue

```diff
@@ -2436,6 +2436,27 @@ export default {
         }
       );
     },
+    
+    sortList(obj) {
+      var sortable = [];
+      for (var key in obj) {
+        if (obj.hasOwnProperty(key)) {
+          sortable.push([key, obj[key]]);
+        }
+      }
+
+      sortable.sort(function (a, b) {
+        var x = a[0].toLowerCase(),
+            y = b[0].toLowerCase();
+        return x < y ? -1 : x > y ? 1 : 0;
+      });
+      
+      var json = JSON.stringify(sortable);
```

Comment:
:fearful: 

You should never manipolate a string to convert one data structure to another.
Please bear in mind that an object doesn't have the concept of order: you need to use and array to preserve the sorting.

Reply:
Yes, I agree with @gsanchietti. You have to declare a new object like this:
```js
var newUserObj = {};
Object.keys(oldUserObject).sort().forEach(function(key) {
  newUserObj[key] = oldUserObject[key];
});
```

than in `newUserObj` should have the object keys ordered.

Reply:
Sorry :D I made changes. Easier than it looks. Thank you so much.